modules/querier/querier.go

```diff
@@ -309,6 +310,11 @@ func (q *Querier) SearchTags(ctx context.Context, req *tempopb.SearchTagsRequest
 		}
 	}
 
+	// Extra hard-coded values (for now)
+	for _, k := range search.GetStaticTags() {
+		uniqueMap[k] = struct{}{}
+	}
```

Comment:
This should only be necessary for the virtual tags right? I'd leave out `status.code` since it will be indexed correctly and actually exists in the ingesters.

WDYT about renaming `GetStaticTags` -> `GetVirtualTags`? That should make it more clear why this is needed here.

Reply:
I did consider naming it `GetVirtualTags` but left it matching `GetxxxTagValues` because they are related.  Based on how we address the other [comment](https://github.com/grafana/tempo/pull/1093#discussion_r740927391) yes I'm on board with renaming/changes in this area.  Also why I left it populating `status.code` (for symmetry).

Reply:
Renamed according to discussion.